### PR TITLE
Problem: the contract paths in integration tests are hard to maintain

### LINKED ISSUE #271: Feat: Add `CONTRACTS` dict for contract paths
**Is your feature request related to a problem? Please describe.**
As you know, all paths of artifact contract json files are fixed. And we always get the path by
```
Path(__file__).parent
        / "contracts/artifacts/contracts/Greeter.sol/Greeter.json"
```
when deploying the contract.

Paths of artifact contract json files are better to be saved/retrieved in a constant value.

**Describe the solution you'd like**
Make a `CONTRACT` dictionary (just like `KEYS`, `ADDRS`) to save all contract paths. The benefits:
1. No need to type the PATH every time we deploy the contract.
2. Make all path consistent.
3. More readable and avoid typing error. 
etc.

### PR DESCRIPTION
**Pull request by damoncro**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

Closes: #271 

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/utils.py [hunk 1]
```diff
@@ -35,6 +35,20 @@
...
+    "Greeter": Path(__file__).parent
+    / "contracts/artifacts/contracts/Greeter.sol/Greeter.json",
+    "ModuleCRC20": Path(__file__).parent.parent
+    / "x/cronos/types/contracts/ModuleCRC20.json",
+    "TestERC20A": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestERC20A.sol/TestERC20A.json",
+    "TestRevert": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestRevert.sol/TestRevert.json",
+    "TestERC20Utility": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestERC20Utility.sol/TestERC20Utility.json",
```
Comment (yihuang) [inline integration_tests/utils.py:50]: share the `Path(__file__).parent / "contracts/artifacts/contracts"` with a global var?

Comment (damoncro) [reply integration_tests/utils.py:50]: Do you mean
```
TEST_CONTRACT_PATH = Path(__file__).parent / "contracts/artifacts/contracts"
CONTRACTS = {
    "Gravity": TEST_CONTRACT_PATH / "Gravity.sol/Gravity.json",
    "Greeter": TEST_CONTRACT_PATH / "Greeter.sol/Greeter.json",
    "ModuleCRC20": Path(__file__).parent.parent
    / "x/cronos/types/contracts/ModuleCRC20.json",
    "TestERC20A": TEST_CONTRACT_PATH / "TestERC20A.sol/TestERC20A.json",
    "TestRevert": TEST_CONTRACT_PATH / "TestRevert.sol/TestRevert.json",
    "TestERC20Utility": TEST_CONTRACT_PATH
    / "TestERC20Utility.sol/TestERC20Utility.json",
}

```

Comment (damoncro) [reply integration_tests/utils.py:50]: Or just one `TEST_CONTRACT_PATH = Path(__file__).parent / "contracts/artifacts/contracts"`?

**Review by thomas-nguy — APPROVED** [review]
lgtm even though I am not sure how using a map make the code more readable :/

looks that it brings more maintenance to me, but its more a style preference

**Comment by damoncro** [comment]
Yes, of course, it would not make the program have big difference, but it is more easy to maintain, and it follows the original design (aka, it does not break the design) just like accessing keys via `KEYS`, accessing addresses via `ADDRS`.

**Review by yihuang — APPROVED** [review]
can you update the title and commit messages follow the C4 patch requirement: https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements, LGTM otherwise.

**Pull request merged**